### PR TITLE
[TASK] Declare TYPO3 v12 and TYPO3 Console v8 compatibility

### LINKED ISSUE #8: TYPO3 Console v8
Currently, this package requires `helhum/typo3-console` v7. It would be great if we could use it with v8 as well.

### PR DESCRIPTION
**Pull request by ohader**
Resolves: #8

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
     "description" : "Execute TYPO3 Console commands on 'composer install'",
     "license": "GPL-2.0-or-later",
     "require": {
-        "php": ">=7.4.1",
-        "helhum/typo3-console": "^7.0",
-        "typo3/cms-composer-installers": "^3.0 || ^4.0"
+        "php": ">=7.4.1 || >=8.1",
```
Comment (helhum) [inline composer.json]: this change is redundant

**Comment by helhum** [comment]
Thanks! Did you check whether everything works after declaring compatibility? I assume it should.

**Comment by ohader** [comment]
I tested on a website that has been upgraded from TYPO3 v11 → v12

**Comment by helhum** [comment]
> I tested on a website that has been upgraded from TYPO3 v11 → v12

Cool, thanks. If you remove the PHP requirement change, I'll merge and release. Thanks again.

File: composer.json (hunk 1 above)
Comment (ohader) [reply composer.json]: Ah, correct... I was thinking in `^7.4.1 || ^8.1`...

Comment (ohader) [reply composer.json]: Reverted the PHP version change

**Pull request merged**